### DISCUSSION TITLE
SRGB off option via websockets  (High CPU Utilization)

### DISCUSSION THREAD

**Discussion by jaggi-mohit** [obsproject/obs-studio#11405] [Development Questions]
Hi Devs,

I am working on multiple instances of obs to achieve multiple UDP outputs. Currently, my CPU utilization is shooting up to 80% with only 2 instances though previously it was up to 90% after activating the hardware acceleration option on input sources it came down to 80 to 85% but still it's too much.

I want to utilize more GPU but even after activating hardware acceleration and using h264_nvenc as video codec it still reaches 12% for two instances. After exploring a bit I found the 'SRGB off' option in the blending method ( which was previously Default ) which makes my CPU utilization a bit lower but it is not available via Websocket APIs, as I have to manage everything via APIs and backend.

any help will be helpful which make my CPU utilization a bit lower

Currently, my single OBS instance architecture looks like this:
![image](https://github.com/user-attachments/assets/339d7246-072e-414e-8863-ea6b26025ca1)

I am switching between different scenes.

System Specs:

AWS Ubuntu 22.04 Instance
16 VCPUs
Tesla T4 GPU Card

Thanks in Advance !!

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Since this is effectively a feature request, might be best to use our [Ideas page](https://ideas.obsproject.com)

Thank you!
